site/blog/2022-11-04.mdx

```diff
@@ -0,0 +1,123 @@
+  'latest',
+  true
+);
+SELECT iasql_apply();
+
+SELECT ecr_build(
+  'https://github.com/iasql/iasql-engine/',
+  (SELECT id
+    FROM repository
+    WHERE repository_name = 'simple-express-repository')::varchar(255),
+  './examples/ecs-fargate/prisma/app',
+  'ghp_XXX', -- replace your github personal access token here
+  'main'
+);
+```
+That's it! Now wait for some time and your app is deployed! While your app is being deployed, let's go through the commands we executed in more depth:
+```sql
+SELECT iasql_install('aws_ecs_simplified');
+```
+- This command installs the `aws_ecs_simplified` high-level module. We– at IaSQL– have created that module to make it easy to deploy containerized apps to ECS. But IaSQL is so flexible that anyone can create their own high-level (and of course, low-level) modules and add it to IaSQL.
+```sql
+INSERT INTO ecs_simplified(app_name, app_port, image_tag, public_ip) VALUES(
+  'simple-express',
+  8088,
+  'latest',
+  true
+);
+```
+- This command creates a new `ecs_simplified` app by inserting a new row into the `ecs_simplified` table. Seems pretty easy, right? But under the hood, it's creating all the necessary resources like load balancers, security groups, IAM roles, etc.
+  - You can manually check the tables to see what resources are being created. For example, looking at the `load_balancer` table you'll see a load balancer named `simple-express-load-balancer` is inserted automatically by running the above insert command.
+```sql
+SELECT iasql_apply();
+```
+- The insert command did create the list for what resources should be created, but they've not yet created on the AWS. You need to call the `iasql_apply()` function to sync the cloud with the data on your database.
+  - For more info on the `iasql_apply()` command, check [this guide](https://iasql.com/docs/apply-and-sync/#apply-and-preview_apply) on how it works.
+- After running the `iasql_apply()` function to completion all the necessary resources are now created on the cloud, so their ARNs are populated. You can verify this by looking at different tables, eg. `iam_role`.
+- To get your load balancer address, you can easily run `SELECT load_balancer_dns FROM ecs_simplified WHERE app_name = 'simple-express'` query and get the URL to access your app.
+- Now the ECS is waiting for an image to be pushed to your ECR repository to run it. You can get the URI for the ECR repository by running the `SELECT repository_uri FROM ecs_simplified WHERE app_name = 'simple-express'` query.
+  - In the next step, we'll automatically build an image for the code in Github repo and then push it to this URI (all through SQL and using another high-level function named `ecr_build`).
+```sql
+SELECT ecr_build(
+  'https://github.com/iasql/iasql-engine/',
+  (SELECT id
+    FROM repository
+    WHERE repository_name = 'simple-express-repository')::varchar(255),
+  './examples/ecs-fargate/prisma/app',
+  'ghp_XXX', -- replace your github personal access token here
```

Comment:
Re-emphasizing here. It could just be `NULL` and then we don't mess with the github token, but can still have an obvious call-out in the comments about adding a github personal access token if it's a private repo.